x-pack/plugins/monitoring/server/alerts/license_expiration_rule.test.ts

```diff
@@ -21,9 +21,11 @@ jest.mock('../lib/alerts/fetch_clusters', () => ({
   fetchClusters: jest.fn(),
 }));
 jest.mock('moment', () => {
-  const moment = function () {};
-  moment.duration = () => ({ humanize: () => 'HUMANIZED_DURATION' });
-  return moment;
+  const actual = jest.requireActual('moment');
+  return {
+    ...actual,
+    duration: () => ({ humanize: () => 'HUMANIZED_DURATION' }),
+  };
```

Comment:
Not sure why exactly, but fully stubbing moment was causing error during the test because some internals of core was using it. Only mocking `duration` fixed it.